spec/features/admin/allocation_chart_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Admin Allocation chart', type: :feature do
+  let(:admin_user) { create(:user, :admin, occupation: :administrative) }
+  let(:company) { admin_user.company }
+
+  before do
+    sign_in(admin_user)
+  end
+
+  describe 'Index' do
+    let(:project) { create(:project, company: company) }
+    let!(:user) { create(:user, allocations: [build(:allocation, project: project, company: company)], company: company) }
+
+    before { visit '/admin/allocation_chart' }
+
+    it 'have table with fields "Nome", "Habilidades", "Cliente", "Alocado até"' do
+      within 'table' do
+        expect(page).to have_text('Nome') & have_text('Habilidades') & have_text('Cliente') & have_text('Alocado até')
```

Comment:
I would suggest adding `aggregate_failures` in here, so in case one of these fields fails we know exactly which one is it. As of now if one of these fields fails the spec will throw a single error without specifying which one is it. https://relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures

Reply:
composable matches already do this